app/models/user.rb

```diff
@@ -1,3 +1,8 @@
 class User < ActiveRecord::Base
   belongs_to :estate
+
+  # TODO: Delete me when the column has dropped
+  def self.columns
+    super.reject { |c| c.name == 'estate_id' }
+  end
```

Comment:
This is a well known pattern to avoid downtime when a column needs to be dropped.

When the migration runs to drop the column and the app servers haven't yet been restarted AR still tries to write to the column because it uses prepared statements. To avoid this short downtime the first step is to tell ActiveRecord to ignore the column and deploy, then drop the column.